src/plugins/cssSnippetManager.tsx

```diff
@@ -0,0 +1,374 @@
+                                                                onClick={() => {
+                                                                    setSnippetsGrouped(snippets => {
+                                                                        snippets[guild_id][channel_id] = snippets[guild_id][channel_id].filter(sn => sn.messageId !== s.messageId);
+
+                                                                        for (const guild of Object.keys(snippets)) {
+                                                                            for (const channel of Object.keys(snippets[guild])) {
+                                                                                console.log(`ayo, Checking ${guild}.${channel}: ${snippets[guild][channel].length}`);
+                                                                                if (!snippets[guild][channel].length) delete snippets[guild][channel];
+                                                                            }
+                                                                            if (!Object.keys(snippets[guild]).length) delete snippets[guild];
+                                                                        }
+
+                                                                        forceUpdate();
+                                                                        setValue(Object.values(snippets).map(channels => Object.values(channels)).flat(2));
+                                                                        return snippets;
+                                                                    });
+                                                                }}
+                                                            >
+                                                                Remove
+                                                            </span>
+                                                        </div>
+                                                        <CodeBlock content={s.code.trim()} lang="css" />
+                                                    </li>
+                                                ))}
+                                            </ul>
+                                        </div>
+                                    );
+                                })}
+                            </li>
+                        );
+                    })
+                }
+            </ul>
+        </ErrorBoundary>
+    );
+};
+
+const SnippetManager = ({ message }: { message: Message; }) => {
+    const forceUpdate = useForceUpdater();
+
+    const channelsTxt: string = Vencord.PlainSettings.plugins.CssSnippetManager.channels || "";
+    const channels = channelsTxt.split(",").map(ch => ch.trim());
+
+    if (!channels.includes(message.channel_id)) return <></>;
+
+    const codeBlocks = parseCodeBlocks(message.content);
+    if (codeBlocks.length !== 1) return <></>;
```

Comment:
```suggestion
    if (codeBlocks.length !== 1) return null;
```
null is the proper return to render nothing!